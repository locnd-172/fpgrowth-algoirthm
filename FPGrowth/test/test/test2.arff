@relation retail

@attribute Beans {true}
@attribute Cheese {true}
@attribute Eggs {true}
@attribute Milk {true}
@attribute Snack {true}
@attribute Yogurt {true}

@data

true,?,true,?,?,true
?,true,?,true,?,true
?,true,true,true,true,true
?,true,true,true,?,true
true,?,true,?,true,?
true,?,?,?,?,true
?,true,?,true,true,true
?,true,?,true,true,true
?,?,?,true,true,?
true,?,true,true,?,true
true,true,?,true,true,?
?,?,true,?,true,true
