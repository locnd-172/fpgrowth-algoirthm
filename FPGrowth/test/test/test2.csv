1,Milk
1,Yogurt
1,Cheese
2,Milk
2,Eggs
2,Yogurt
2,Cheese
3,Snack
3,Beans
3,Eggs
4,Beans
4,Yogurt
5,Milk
5,Cheese
5,Snack
5,Yogurt
6,Snack
6,Milk
6,Yogurt
6,Cheese
7,Snack
7,Milk
8,Milk
8,Yogurt
8,Eggs
8, Beans
9,Beans
9,Snack
9,Cheese
9,Milk
10,Yogurt
10,Eggs
10,Snack
11,Yogurt
11,Eggs
11,Beans
12,Milk
12,Yogurt
12,Cheese
12,Eggs
12,Snack
