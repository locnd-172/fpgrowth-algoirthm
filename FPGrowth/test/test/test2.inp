Beans Eggs Yogurt 
Cheese Milk Yogurt 
Cheese Eggs Milk Snack Yogurt 
Cheese Eggs Milk Yogurt 
Beans Eggs Snack 
Beans Yogurt 
Cheese Milk Snack Yogurt 
Cheese Milk Snack Yogurt 
Milk Snack 
Beans Eggs Milk Yogurt 
Beans Cheese Milk Snack 
Eggs Snack Yogurt 
