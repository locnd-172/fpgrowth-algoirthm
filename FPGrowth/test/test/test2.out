Eggs:6
Milk:8
Beans:5
Snack:7
Cheese:6
Yogurt:9
Milk Snack:5
Cheese Milk:6
Eggs Yogurt:5
Milk Yogurt:6
Cheese Yogurt:5
Cheese Milk Yogurt:5
